_data/simple-icons.json

```diff
@@ -10541,7 +10541,7 @@
         },
         {
             "title": "Reason Studios",
-            "hex": "FFFFFF",
+            "hex": "000000",
```

Comment:
I think a shade of yellow like `#F9CB61` might be a better choice. What do you think?